### ISSUE TITLE
Bayesian Semantic Instance Segmentation in Open Set World

### ISSUE THREAD

**Issue by aljosaosep** [aljosaosep/cool_papers#5]
**Bayesian Semantic Instance Segmentation in Open Set World**
_Venue:_ ECCV'18
_Authors:_ Trung Pham, Vijay Kumar B G, Thanh-Toan Do, Gustavo Carneiro, and Ian Reid
_Keywords:_ Instance segmentation, Open -set, Bayesian
_Baselines:_ Custom baseline.
_Tasks:_ Semantic instance segmentation in open-set conditions: input images contain both, known and unknown object classes. We don't have annotations for all object classes.
_Evaluation:_ Train on COCO, eval on NYU. NYU has annot. for 894 classes (781 objects, 113 stuff). 60 classes overlap with COCO.
_Brief: Marriage of new stuff (Faster/Mask R-CNN), supervised and traditional segmentation methods. Use several traditional cues (contours, edges, priors); nice (Bayesian) formulation that can incorporate cues from pre-trained detector/instance segmentation method and output dense MAP estimate. MAP inference done using simulated annealing. Nice combination of new stuff and traditional methods.
_Contributions:_ Probably the first open-set instance segmentation method that works well. Nice idea: learn supervised what you can and infer the rest.

![image](https://user-images.githubusercontent.com/10114300/45705915-258a6b00-bb7b-11e8-99ac-7385846096c8.png)

![image](https://user-images.githubusercontent.com/10114300/45706241-093afe00-bb7c-11e8-93e8-ae5fa96149c4.png)

![image](https://user-images.githubusercontent.com/10114300/45706257-15bf5680-bb7c-11e8-9f7d-f976fa2ebcf3.png)

![image](https://user-images.githubusercontent.com/10114300/45706344-48694f00-bb7c-11e8-8f9e-1990279ea1b6.png)

![image](https://user-images.githubusercontent.com/10114300/45706200-ea3c6c00-bb7b-11e8-93e7-caf71bdd1dd2.png)

![image](https://user-images.githubusercontent.com/10114300/45706204-ef012000-bb7b-11e8-8b41-fdc0c6912b48.png)

**Comment by theRealSuperMario** [comment]
Do you know if there is a repository with the code maybe? Otherwise, I think I will work on a reimplementation myself.

**Comment by YangLiu14** [comment]
> Do you know if there is a repository with the code maybe? Otherwise, I think I will work on a reimplementation myself.

Hey man, have you succeeded in reimplementing the paper? If so, I'd be really interested in that.
